chrome_manifest_v3/views/class_popupviewforms.js

```
@@ -0,0 +1,958 @@
+        case 'signout':
+          $status.html('Done');
+          this.app.events.fire('onRequestDeauthorizeTrello');
+          break;
+        case 'reload':
+          this.parent.forceSetVersion(); // Sets value for version if needing update
+          $status.html('Reloading');
+          window.location.reload(true);
+          break;
+        case 'clearCacheNow':
+          $status.html('Clearing');
+          let hash = {};
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)